src/newrelic/collector/client.go

```diff
@@ -402,26 +428,13 @@ func (c *clientImpl) Execute(cmd *RpmCmd, cs RpmControls) RPMResponse {
 	// Create the JSON payload
 	data, err := cs.Collectible.CollectorJSON(false)
 	if nil != err {
-		return RPMResponse{Err: err}
+		return NewRPMResponseError(err)
 	}
 	cmd.Data = data
 
-	var audit []byte
-	if log.Auditing() {
-		audit, err = cs.Collectible.CollectorJSON(true)
-		if nil != err {
-			log.Errorf("unable to create audit json payload for '%s': %s", cmd.Name, err)
-			audit = cmd.Data
-		}
-		if nil == audit {
-			audit = cmd.Data
-		}
-	}
-
 	url := cmd.url(false)
 	cleanURL := cmd.url(true)
 
-	log.Audit("command='%s' url='%s' payload={%s}", cmd.Name, url, audit)
```

Comment:
Is logging to audit file removed because log file will already have the same info but with URL obfuscated? However, isn't the purpose of the audit file to include all communication with the collector?

Reply:
Yes, I removed it because it is already included in the log file. I made the decision to not allow the full URL ever, in order to ensure full compliance.

Reply:
Then wouldn't:
```
log.Audit("command='%s' url='%s' payload={%s}", cmd.Name, cleanUrl, audit)
```
be better?

Reply:
re-added in [93d2ddc](https://github.com/newrelic/newrelic-php-agent/pull/795/commits/93d2ddc631d1f7d0d1c9d9163b8eb66130961be5)